test/goldens/scope/MORESEMI3.broken

```diff
@@ -0,0 +1,5 @@
+(Left
```

Comment:
The `bar` in the semi-join isn't exposed to outside conditions.   I honestly have no idea whether it's supposed to be :-P